packages/openapi-v3-types/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@loopback/openapi-v3-types",
+  "version": "1.0.0-alpha.1",
+  "description": "TypeScript type definitions for OpenAPI Specifications.",
+  "engines": {
+    "node": ">=8"
+  },
+  "dependencies": {
+    "openapi3-ts": "^0.7.0"
+  },
+  "devDependencies": {
+    "@loopback/build": "^4.0.0-alpha.13"
+  },
+  "scripts": {
+    "build": "lb-tsc es2017",
+    "build:apidocs": "lb-apidocs",
+    "clean": "lb-clean loopback-openapi-v3-types*.tgz dist package api-docs",
```

Comment:
noticed that `loopback-openapi-v3-types*.tgz` from this line and `loopback-v3-types*.tgz` were different. Is this intentional?

Reply:
@shimks I followed the examples like https://github.com/strongloop/loopback-next/blob/master/packages/core/package.json#L12 to adjust the clean command, so I assume the convention here is `loopback` + `<package name>`, but not sure, I will double check it.